integration-tests/realm-apps/pv-int-tests/services/mongodb/rules/default.json

```
@@ -0,0 +1,14 @@
+{
+	"roles": [{
+			"name": "defaultRole",
```

Or is this a new "default" name? Also - I'd expect this to match the name of the file to match all the other config files.
```suggestion
			"name": "default",
```

---

Hmm...I guess it doesn't make a big difference, but I can make the change to make it consistant.

---

I renamed the files instead